polynote-kernel/src/test/scala/polynote/kernel/lang/python/PythonInterpreterSpec.scala

```diff
@@ -295,6 +295,29 @@ class PythonInterpreterSpec extends FlatSpec with Matchers with KernelSpec {
     }
   }
 
+  it should "properly handle imports in local scopes" in {
+    assertPythonOutput(
+      """
+        |import math
+        |
+        |def func(x):
+        |    result = math.sin(x)  # create a local var to make sure it doesn't appear in the outputs
```

Comment:
@aishfenton This tests the local var concern you had and looks good! We don't see `result` in the cell output 😄

Reply:
👍